src/include/migraphx/dynamic_loader.hpp

```diff
@@ -45,6 +59,8 @@ struct dynamic_loader
 
     dynamic_loader(const std::vector<char>& buffer);
 
+    static void check_load_error(bool flush = false);
```

Comment:
This function should just be in the .cpp file.

Reply:
done.